extensions/shared/upgrade-nudge/index.jsx

```diff
@@ -8,7 +8,7 @@ import { Button } from '@wordpress/components';
 import { compact, get, startsWith } from 'lodash';
 import { compose } from '@wordpress/compose';
 import { withDispatch, withSelect } from '@wordpress/data';
-import { Warning } from '@wordpress/editor';
+import { Warning } from '@wordpress/block-editor';
```

Comment:
Won't be an issue on .com where we have the latest block editor but just noting that for jetpack self-hosted sites `@wordpress/block-editor` isn't available in all WP versions we support.

Reply:
Resolved by 62221b9 and cbe7dd8.